api/src/test/java/org/apache/iceberg/expressions/TestStrictMetricsEvaluator.java

```diff
@@ -95,7 +95,7 @@
           .put(12, 0L)
           .put(13, 1L)
           .build(),
-  // nan value counts
+      // nan value counts
```

Comment:
There are multiple maps being build above and below this that have a comment like this, but this comment doesn't align with the others.